lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -214,7 +214,7 @@ sub download_asset {
     }
     else {
         # Clean up after ourselves. Probably won't exist, but just in case
-        OpenQA::Utils::log_error("Downloading failed! Deleting files");
+        OpenQA::Utils::log_error("Download of $url to $assetpath failed! Deleting files.");
```

Comment:
Good improvement to better understand the error message without context. Maybe you can change "Deleting files" to something less confusing ("Download failed but it still gets cleared?! Are there more files he wants to delete?") like: "Deleting maybe leftover files in $assetpath".